package.json

```diff
@@ -31,6 +31,6 @@
     "test262-parser-runner": "^0.5.0"
   },
   "dependencies": {
-    "acorn-private-class-elements": "^0.1.1"
+    "acorn-private-class-elements": "^0.2.0"
```

Comment:
Hrm.  Scanning over this change again, I've noticed a discrepancy between the NPM release tags and GitHub releases for `acorn-private-class-elements`:

[NPM](https://www.npmjs.com/package/acorn-private-class-elements) contains `0.2.0` but not `0.2.2`.
[GitHub](https://github.com/acornjs/acorn-private-class-elements/tags) contains `0.2.2` but not `0.2.0`.

Until the correct version(s) are determined, it probably makes sense to pause on merging this.

Reply:
This is now resolved - `0.2.0` is the correct version ([ref](https://github.com/acornjs/acorn-private-class-elements/issues/12#issuecomment-616384654))